backend/create_dummy_data.py

```diff
@@ -77,6 +81,7 @@
     "grade": "Intermediate",
     "year": 2019,
     "cover_url": "https://images-na.ssl-images-amazon.com/images/I/91qAL9ZLi9L.jpg",
+    "published": True,
```

Comment:
this change really caught my attention :100:

Reply:
nice